Custom_filter_Positive_numbers_Timer.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime
+
+
+def my_filter(func, iterable):
```

Comment:
I would chose more distinct variable names  and don't forget type hints;
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/